quill-finagle-postgres/src/main/scala/io/getquill/context/finagle/postgres/FinaglePostgresDecoders.scala

```diff
@@ -42,41 +45,21 @@ trait FinaglePostgresDecoders {
   implicit val stringDecoder: Decoder[String] = decoderDirectly[String]
   implicit val bigDecimalDecoder: Decoder[BigDecimal] = decoderDirectly[BigDecimal]
   implicit val booleanDecoder: Decoder[Boolean] = decoderDirectly[Boolean]
-  implicit val byteDecoder: Decoder[Byte] = decoder[Byte] {
+  implicit val byteDecoder: Decoder[Byte] = decoder[Short, Byte] {
     case v: Short => v.toByte
   }
   implicit val shortDecoder: Decoder[Short] = decoderDirectly[Short]
-  implicit val intDecoder: Decoder[Int] =
-    decoder[Int] {
-      case v: Int  => v
```

Comment:
Currently the conversion of numeric types differs between modules, we'd better make them follow jdbc standards or something else